src/transformers/models/whisper/tokenization_whisper.py

```diff
@@ -231,27 +361,57 @@ def bpe(self, token):
         self.cache[token] = word
         return word
 
-    # Copied from transformers.models.gpt2.tokenization_gpt2.GPT2Tokenizer.build_inputs_with_special_tokens with GPT2 -> Whisper
-    def build_inputs_with_special_tokens(self, token_ids_0, token_ids_1=None):
-        if self.add_bos_token:
-            bos_token_ids = [self.bos_token_id]
-        else:
-            bos_token_ids = []
+    def set_bos_sequence(self, language=None, task=None, predict_timestamps=False):
```

Comment:
Also wondering if we should maybe do 
`self.language = language if language is not None`, that you can also set the prefix tokens by calling this function once. WDYT? Or we can have a real `set_prefix_tokens` somewhere else.

Reply:
It's probably unlikely someone will load a tokenizer and want to set it to two different languages -> setting the tokenizer language is just for fine-tuning. In most cases, this will involve just one language.

But I'll add the functionality so it's there though as it's few extra lines of code! I think `self.language = ... ` is preferable here to override the language for the tokenizer.

Reply:
Resolved in https://github.com/huggingface/transformers/pull/19921/commits/e9cb54612d5218365223546c32fa1d36145a6e75